client/src/features/login/LogIn.tsx

```diff
@@ -8,57 +8,71 @@ import { FaUnlockAlt } from 'react-icons/fa';
 
-const LogIn = () => {
+const LogIn: React.FC = () => {
   const [inputs, setInputs] = useState<ISingUpLogIn>({
     email: '',
     password: '',
   });
   const [isVisible, setIsVisible] = useState<boolean>(false);
+  const [errMsgFromServer, setErrMsgFromServer] = useState<string>('');
+  const navigate = useNavigate();
 
   const loginWithEmailAndPassword = async (
     e: React.FormEvent<HTMLButtonElement>
   ) => {
     e.preventDefault();
     try {
       console.log('clicked');
-      await axios.post('http://localhost:5000/login', inputs, {
+      const response = await axios.post('http://localhost:5000/login', inputs, {
         withCredentials: true,
       });
-      console.log('logged in successfully!');
-      setInputs({
-        email: '',
-        password: '',
-      });
-    } catch (error) {
+      navigate('/board');
+      console.log(response.data.message);
+    } catch (error: any) {
       console.log(error);
+      if (error.response) {
+        console.error(error.response.data.message);
+        setErrMsgFromServer(error.response.data.message);
+      } else {
+        console.error(error);
+        setErrMsgFromServer('Sorry, unexpected error occured');
+      }
     }
   };
+
+  const handleOnChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target;
+    errMsgFromServer.length !== 0 && setErrMsgFromServer('');
+    setInputs({ ...inputs, [name]: value });
+  };
   return (
     <main className="flex w-screen h-screen bg-primary_bg_color text-primary_title_color">
       <LeftPart />
       <section className="flex justify-center my-auto h-9/12 w-1/2 tablet_l_max:w-full ">
         <div className="flex flex-col justify-end h-full w-7/12 mobile_xl_max:w-9/12">
-          <h2 className="flex-initial text-4xl mb-2 mobile_l_max:text-3xl">
+          <h2 className="flex-initial text-4xl mb-3 mobile_l_max:text-3xl">
             Login
           </h2>
-          <p className="flex-initial mb-12 mobile_l_max:text-sm">
-            Not registered yet?{' '}
+          <div className="mb-12 flex mobile_m_max:flex-col">
+            <p className="flex-initial mr-1 mobile_l_max:text-sm">
+              Not registered yet?
+            </p>
             <Link to="/signup" className="underline font-bold">
               Create an Account
             </Link>
-          </p>
+          </div>
           <form action="" className="flex flex-col w-full">
             <label htmlFor="email" className="text-lg mb-2">
               Email Address
             </label>
             <div className="relative">
               <input
+                id="email"
                 type="email"
-                className="mb-8 h-10 text-xl w-full pl-8"
-                onChange={(e) =>
-                  setInputs({ ...inputs, email: e.target.value })
-                }
+                className="mb-8 h-10 text-xl w-full pl-8 rounded "
+                onChange={(e) => handleOnChange(e)}
```

Comment:
```suggestion
                onChange={handleOnChange}
```
The same goes for line 87 and 132.